### PR TITLE
Update Rendezvous() description

### LINKED ISSUE #676: Preconnection.Resolve() needs work
In §6.3 we have:

>    If the endpoints are suspected to be behind a NAT, Rendezvous() can
>    be initiated using Local and Remote Endpoints that support a method
>    of discovering NAT bindings such as Session Traversal Utilities for
>    NAT (STUN) [RFC8489] or Traversal Using Relays around NAT (TURN)
>    [RFC5766].  In this case, the Local Endpoint will resolve to a
>    mixture of local and server reflexive addresses.  The Resolve()
>    action on the Preconnection can be used to discover these bindings:
>    ```[]Preconnection := Preconnection.Resolve()```
>    The Resolve() call returns a list of Preconnection Objects, that
>    represent the concrete addresses, local and server reflexive, on
>    which a Rendezvous() for the Preconnection will listen for incoming
>    Connections.  These resolved Preconnections will share all other
>    Properties with the Preconnection from which they are derived, though
>    some Properties may be made more-specific by the resolution process.

This doesn't work, because `Preconnection.Rendezvous()` takes only a single `Preconnection`. The `Resolve()` action on a `Preconnection` should rather be on a `LocalEndpoint` or `RemoteEndpoint`.

### PR DESCRIPTION
**Pull request by csperkins**
Closes #676

### CODE REVIEW

File: draft-ietf-taps-interface.md [hunk 1]
```diff
@@ -1508,53 +1508,58 @@ Rendezvous() Action:
...
-are then configured on the Preconnection, allowing the Rendezvous() Action to be
-initiated.
-
-The Rendezvous() Action returns a Connection object. Once Rendezvous() has been
-called, any changes to the Preconnection MUST NOT have any effect on the
-Connection. However, the Preconnection can be reused, e.g., for Rendezvous of
-another Connection.
+in the presence of NATs will configure the Preconnection object with at least
+one a Local Endpoint that supports NAT binding discovery. It will then Resolve()
+the Preconnection, and pass the resulting list of Local Endpoints candidates to
```
Comment (mwelzl) [inline draft-ietf-taps-interface.md:1546]:
```suggestion
the Preconnection, and pass the resulting list of Local Endpoint candidates to
```

File: draft-ietf-taps-interface.md [hunk 2]
```diff
@@ -1566,35 +1571,21 @@ transport-layer connection is established; for Connectionless transports,
 it occurs when the first Message is received from the Remote Endpoint. The
 resulting Connection is contained within the RendezvousDone<> Event, and is
 ready to use as soon as it is passed to the application via the Event.
+Changes made to a Preconnection made after Rendezvous() has been called do
```
Comment (mwelzl) [inline draft-ietf-taps-interface.md:1574]:
```suggestion
Changes made to a Preconnection after Rendezvous() has been called do
```

**Review by mwelzl — COMMENTED** [review]
LGTM, I made two tiny text suggestions; and thanks!

File: draft-ietf-taps-interface.md (hunk 1 above)
Comment (csperkins) [reply draft-ietf-taps-interface.md:1546]: Fixed.

File: draft-ietf-taps-interface.md (hunk 2 above)
Comment (csperkins) [reply draft-ietf-taps-interface.md:1574]: Fixed

**Review by mwelzl — APPROVED** [review]

**Review by tfpauly — APPROVED** [review]

**Pull request merged**